ApiSamples/Samples/QueryBuildingInformation.m

```diff
@@ -57,21 +58,19 @@ - (void)mapView:(WRLDMapView *)mapView didReceiveBuildingInformationForHighlight
 
     for (WRLDBuildingContour* contour in buildingInformation.contours)
     {
-        CLLocationCoordinate2D coordinates[contour.pointCount+1];
+        CLLocationCoordinate2D* coordinates = malloc((contour.pointCount+1) * sizeof(CLLocationCoordinate2D));
```

Comment:
Not sure if malloc / free is idiomatic here, we don't use them elsewhere in the examples. Can we use a container type here?

Reply:
Why use the heap for this tmp var ? Allocate and free in a loop is usually not recommended. 
It should be possible to pass the `coordinates` array from the stack to the `getPoints` function without allocating ?